action_server/src/action_server/actions/action.py

```diff
@@ -115,8 +129,11 @@ def _start(self):
         raise NotImplementedError
 
     def cancel(self):
+        """
+        Cancels the execution of the action.
+        """
         rospy.loginfo("Canceling executing of action {}.".format(self.get_name()))
-        return self._cancel()
+        self._cancel()
 
     def _cancel(self):
         raise NotImplementedError
```

Comment:
I usually do ``raise NotImplementedError("Implement me in " + str(type(self)))`` so I get the name of the class not implementing the method. For a module ``x.py``:
``` python
class A(object):
  def x(self):
    raise NotImplementedError
  def y(self):
    raise NotImplementedError("Implement me in " + str(type(self)))

class B(A):
    pass
```
With calls:
```
>>> import x
>>> b = x.B()
>>> b.x()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "x.py", line 3, in x
    raise NotImplementedError
NotImplementedError
>>> b.y()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "x.py", line 5, in y
    raise NotImplementedError("Implement me in " + str(type(self)))
NotImplementedError: Implement me in <class 'x.B'>

```
```